build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.Unix.cs

```diff
@@ -18,7 +18,7 @@ partial class Urls
 			// This is the "public" url that we really should be using, but it keeps failing with:
 			// AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: RevocationStatusUnknown
 			// For now we'll grab it directly from GitHub
-			// public static readonly Uri DotNetInstallScript = new Uri ("https://dot.net/v1/dotnet-install.sh");
+			// public static readonly Uri DotNetInstallScript = new Uri ("https://builds.dotnet.microsoft.com/dotnet/scripts/v1/dotnet-install.sh");
 			public static readonly Uri DotNetInstallScript = new Uri ("https://raw.githubusercontent.com/dotnet/install-scripts/refs/heads/main/src/dotnet-install.sh");
```

Comment:
I wonder if we should try reverting this too and checking if the issues still exist.

Reply:
Yes, I would say change it to:
```suggestion
			public static readonly Uri DotNetInstallScript = new Uri ("https://builds.dotnet.microsoft.com/dotnet/scripts/v1/dotnet-install.sh");
```